periphery/src/LenderLens.sol

```diff
@@ -18,13 +17,28 @@ contract LenderLens {
             uint256 utilization,
             uint256 inventory,
             uint256 totalBorrows,
-            uint256 totalSupply
+            uint256 totalSupply,
+            uint8 reserveFactor
         )
     {
         asset = lender.asset();
         (, inventory, totalBorrows, totalSupply) = lender.stats();
 
-        if (inventory != 0) utilization = Math.mulDiv(1e18, totalBorrows, inventory);
+        if (inventory > 0) utilization = (1e18 * totalBorrows) / inventory;
         interestRate = lender.rateModel().getYieldPerSecond(utilization, address(lender));
+        reserveFactor = lender.reserveFactor();
```

Comment:
Return reserveFactor as part of `readBasics`. I checked frontend code; since this is at the end of the array it shouldn't break any of our current stuff. Just makes it possible to simplify calls in the future.